pkgs/games/cataclysm-dda/git.nix

```diff
@@ -2,9 +2,9 @@
 , tiles ? true, Cocoa
 , debug ? false
 , useXdgDir ? false
-, version ? "2019-11-22"
-, rev ? "a6c8ece992bffeae3788425dd4b3b5871e66a9cd"
-, sha256 ? "0ww2q5gykxm802z1kffmnrfahjlx123j1gfszklpsv0b1fccm1ab"
+, version ? "2020-12-09"
```

Comment:
Do more recent builds fail?

Reply:
I didn't check, I decided to use this build because this seems to be the latest stable build they have available. The more recent builds that I saw seemed to be experimental and are changed frequently.